tests/nircam/nircam_sensitivity_lw.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 import astropy.io.fits as fits
 from verification_tools import calc_limits
-from verification_tools import fudge_throughput as ft
+#from verification_tools import fudge_throughput as ft
```

Comment:
Is there a reason this is commented out instead of just deleted?

Reply:
Initially, to see if it was needed but broken. I have removed it.